article.html

```diff
@@ -7,37 +7,76 @@
 
 	<body>
 		<header>
-			<p>This will be the nav</p>
+			<a href="index.html" class="logo">Skinclusive</a>
+			<nav name="top">
+				<ul>
+					<li><a href="quiz.html">skin type quiz</a></li>
+					<li class="dropdown"><a href="all-products.html">products &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin products</a>
+							<a href="#">dry skin products</a>
+							<a href="#">combo skin products</a>
+							<a href="#">sensitive skin products</a>
+						</div>
+					</li>
+					<li class="dropdown"><a href="article-list.html">articles &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin articles</a>
+							<a href="#">dry skin articles</a>
+							<a href="#">combo skin articles</a>
+							<a href="#">sensitive skin articles</a>
+						</div>
+					</li>
+				</ul>
+			</nav>
 		</header>
 		
 		<div class="wrapper">
             <!-- This div is for the template for all articles -->
             <div class="article">
                 <h1>Article header here</h1>
-                <h2>Author and publish date here</h2>
+                <h2>Skinclusive | DD/MM/YYYY</h2>
 
                 <div class="video">
                     <p>Placeholder for a video</p>
                 </div>
-                <p>Article text will go here. Relevant images will also be spread out through this page</p>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor 
+                    incididunt ut labore et dolore magna aliqua. Curabitur vitae nunc sed velit dignissim sodales ut.
+                    Blandit massa enim nec dui nunc mattis. Varius quam quisque id diam vel quam elementum.
+                    Dui ut ornare lectus sit amet. Elementum eu facilisis sed odio morbi quis commodo odio. 
+                    Risus sed vulputate odio ut enim blandit volutpat maecenas. Nisi est sit amet facilisis magna. 
+                    At in tellus integer feugiat scelerisque varius morbi enim. Eget lorem dolor sed viverra. 
+                    Aliquet eget sit amet tellus cras. Et malesuada fames ac turpis egestas maecenas pharetra 
+                    convallis posuere. Faucibus et molestie ac feugiat sed. Ipsum suspendisse ultrices gravida 
+                    dictum fusce ut.</p>
+                
+                <div class="content-image">Images will be scattered where relevant</div>

```

Comment:
I also added a blank placeholder image by default, so that if I want to add a relevant image in an article, I can simply replace the placeholder with the actual image, or just take out this line of code entirely if undesired.